DQMOffline/Trigger/python/TopMonitoring_cff.py

```diff
@@ -2,6 +2,15 @@
 
 from DQMOffline.Trigger.TopMonitor_cfi import hltTOPmonitoring
 
+from Configuration.Eras.Modifier_run2_HLTconditions_2018_cff import run2_HLTconditions_2018
+from Configuration.Eras.Modifier_run2_HLTconditions_2017_cff import run2_HLTconditions_2017
+from Configuration.Eras.Modifier_run2_HLTconditions_2016_cff import run2_HLTconditions_2016
+from Configuration.StandardSequences.Eras import eras 
```

Comment:
@skonstantinou it seems this last `import` is unnecessary; if so, please remove it.